app/src/main/java/org/luxoft/sdl_core/MainActivity.java

```diff
@@ -299,48 +297,7 @@ public void onRequestPermissionsResult(int requestCode, String[] permissions, in
     }
 
     private void onSdlStopped() {
```

Comment:
@VadymLuchko probably we can remove an empty function?

Reply:
fixed in f874c10935f6b28d6931e867ca35d2992f8b0469